### ISSUE TITLE
Problems with PUT IMAGE(S) under CPC 664

### ISSUE THREAD

**Issue by ldir-hector** [spotlessmind1975/ugbasic#541]
On Amstrad CPC,
with the instruction PUT IMAGE <img> FRAME <num> AT x,y WITH TRANSPARENCY if x > 256 then the image is displayed at the left of the screen, even if it was configured to be in 320*200 mode.

The following gode illustrate it : 

```
BITMAP ENABLE(320,200,4)
CLS
COLOR BORDER RGB(0,0,0)

caracteresCPC := LOAD IMAGES("symbolesCPC_mode_1.png") FRAME SIZE (8,8)
cpt=0
x=300
y=0
FOR i=32 TO 255
	temp$=CHR$(i)
	CLS
	PUT IMAGE caracteresCPC FRAME (ASC(temp$)-32) AT x,y WITH TRANSPARENCY
	WAIT KEY
NEXT
```
![image](https://user-images.githubusercontent.com/118670680/210144873-624eb2c4-b5ef-4bbe-91d6-c289890162a3.png)

**Comment by ldir-hector** [comment]
[asc_exemple_bug.cpc.dsk.zip](https://github.com/spotlessmind1975/ugbasic/files/10327569/asc_exemple_bug.cpc.dsk.zip)
![symbolesCPC_mode_1_v2](https://user-images.githubusercontent.com/118670680/210145573-14c23cda-e59a-49d4-965b-1c54bf3d84cc.png)
![symbolesCPC_mode_1](https://user-images.githubusercontent.com/118670680/210145577-2cbc6a52-41bb-4393-9b0c-b2bfb53e33ba.png)

**Comment by ldir-hector** [comment]
I first thought that was a problem with my image as all symbols weren't on the same line, but it does the same with both files.

**Comment by spotlessmind1975** [comment]
Fixed and released on **COLDFIX v1.12.1 [rev. 20230101.1503]**.
